starters/servers/netlify/README.md

```diff
@@ -0,0 +1,35 @@
+## Netlify
+
+This site is configured to deploy to [Netlify Edge Functions](https://www.netlify.com/products/edge/), which means it will be rendered at an edge location near to your users.
```

Comment:
Minor suggestion to use the "starter" term to match Qwik's terminology.
```suggestion
This starter site is configured to deploy to [Netlify Edge Functions](https://www.netlify.com/products/edge/), which means it will be rendered at an edge location near to your users.
```

Reply:
Also, we use "this site" and "your site" interchangeably in the README. I've added comments to update to "this starter site" and "your site" as appropriate. I think we want to make a clear distinction between the starter site project and the site users create.